storage/mysql/tree_storage.go

```diff
@@ -270,13 +270,13 @@ func (t *treeTX) storeSubtrees(subtrees []*storagepb.SubtreeProto) error {
 	// TODO(al): probably need to be able to batch this in the case where we have
 	// a really large number of subtrees to store.
 	args := make([]interface{}, 0, len(subtrees))
+	glog.Warningf("writing %d subtrees", len(subtrees))
```

Comment:
No. I'll take it out. There's not enough logging to debug logic issues so maybe some should go back as log.V() later.